lib/signing.ts

```diff
@@ -67,6 +81,8 @@ export class SignNuGetWithSigner extends Construct implements ISigner {
       SIGNING_BUCKET_NAME: props.signingBucket.bucketName,
       SIGNING_LAMBDA_ARN: props.signingLambda.functionArn,
       ACCESS_ROLE_ARN: props.accessRole.roleArn,
+      SIGNER_PROFILE_NAME: props.signerProfileName ?? 'profile-name',
+      SIGNER_PROFILE_OWNER: props.signerProfileOwner ?? 'profile-owner',
```

Comment:
i think if not specified these env variables should be undefined. Any reason this is not the case?

Reply:
I don't have a specific reason besides thinking maybe we should try for starting the signing job with profile-name and profile-owner if nothing is explicitly provided. I can update this to make it undefined if nothing is provided.

Reply:
This has been updated.